README.md

```diff
@@ -16,13 +16,13 @@ To use N-API in a native module:
   1. Add a dependency on this package to `package.json`:
 ```json
   "dependencies": {
-    "node-addon-api": "github:nodejs/node-addon-api",
+    "node-addon-api": "github:nodejs/node-api",
```

Comment:
I was expecting this repo would actually get renamed to `node-addon-api`. @mhdawson what do you think?